Assets/Scripts/UI/WarningText.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class WarningText : MonoBehaviour
+{
+    private enum alphaValue
+    {
+        SHRINKING,
+        GROWING,
+    }
+
+    private alphaValue currentAlphaValue;
+    [SerializeField] private float CommentminAlpha;
+    [SerializeField] private float CommentmaxAlpha;
+    [SerializeField] private float CommentCurrentAlpha;
+    [SerializeField] private bool flashText;
+
+    [SerializeField] private Text MyText;
+
+    void Start()
+    {
+        CommentminAlpha = 0f;
+        CommentmaxAlpha = 1.0f;
+        CommentCurrentAlpha = 1.0f;
+        currentAlphaValue = alphaValue.SHRINKING;
+    }
+
+    void Update()
+    {
+        if (flashText == true)
+        {
+            alphaComments();
+        }
+        
+    }
+
+    public void SetIsAllowedToFlash(bool textFlash)
+    {
+        flashText = textFlash;
+    }
```

Comment:
```suggestion
    public void SetIsAllowedToFlash(bool targetValue)
    {
        isAllowed to flash = targetValue;
    }
```

De bool "flashText" heeft geen werkwoord, dus je weet niet wat het presiec in houd. Is het een normaal stukje tekst? Kan het nu flashen?